_episodes/04-data-types-and-format.md

```diff
@@ -15,7 +15,12 @@ objectives:
     - "Analyze datasets having missing/null values (NaN values)."
     - "Write manipulated data to a file."
 keypoints:
-    - "FIXME"
+    - Python uses two main data types text and numeric, these have a different names in basic Python and Pandas
```

Comment:
1. I'm not sure what this keypoint means. The "main" data types in Python are numbers, sequences, sets, mappings.
2. There is no "basic" Python, just Python
3. One can't compare Python and Pandas as these are of different kind (Pandas is a Python package). You can say that Pandas does something differently than Python (if that's what you want to say here).

Reply:
keypoints are addressing this episode https://datacarpentry.org/python-ecology-lesson/04-data-types-and-format/index.html

please see suggestion from the text